zerver/openapi/zulip.yaml

```diff
@@ -10716,13 +10716,20 @@ paths:
             type: string
           example: Python
           required: true
-        - name: url_prefix
+        - name: url_template
           in: query
           description: |
-            The url prefix for the playground.
+            The [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html)
+            compliant URL template for the playground. The template should
+            contain exactly one variable named `code`, which determines how the
+            extracted code should be substituted in the playground URL.
+
+            **Changes**: New in Zulip 8.0 (feature level 196). This replaced
+            the `url_prefix` parameter, which was the URL where the extracted
+            code would be appended to.
```

Comment:
Let's spell this out a bit more precisely: "parameter, which was used to construct URLs by just concatenating `url_prefix` and `code`."